### ISSUE TITLE
Mundo invertido

### ISSUE THREAD

**Issue by raquel0685** [digitalinnovationone/semana-frontend-mundo-invertido#7]
Boa noite!

Vocês irão disponibilizar o código para inverter as divs do mundo invertido? Achei que era na live de JavaScript, mas não foi... é na 3?

Obrigada!

**Comment by githubwanderson** [comment]
Boa noite Raquel.

A inversão foi feita na aula 01 depois do CSS.

**Comment by raquel0685** [comment]
Não, a Michelle falou que ia ser feito com JS e só colocou a música.

**Comment by raquel0685** [comment]
Nossa, o meu não estava girando, aí fui conferir o código e havia digitado errado... foi mal! :grin: :flushed:
Obrigada! :kissing_heart:

**Comment by Ritt3r** [comment]
O meu era o classlist, que o L estava minúsculo kkkkk
